['Outputs do not match', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpsldw7avy/solution_3.py", line 46, in <module>\n    print(Solution().maximumEnergy(energy = [5, 5, 5, 5, 5, 5, 5, 5, 5, 5], k = 1))\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpsldw7avy/solution_3.py", line 44, in maximumEnergy\n    helper(m - 2, 0)\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpsldw7avy/solution_3.py", line 35, in helper\n    z[x] = energy[x] + z[q[0]]\nIndexError: deque index out of range\n']